files/install_amp.sh

```diff
@@ -39,55 +38,28 @@ echo "Restarting Syslog"
 sudo systemctl restart rsyslog
 
 echo "Install Java"
-sudo sh -c "export DEBIAN_FRONTEND=noninteractive; apt-get install --yes openjdk-${JAVA_VERSION}-jre-headless"
-sudo sed -i '/assistive_technologies/ s/^#*/#/' /etc/java-8-openjdk/accessibility.properties
+sudo apt-get install -y default-jre-headless
 
 echo "Install AMP"
-tar zxf cloudsoft-amp-karaf.tar.gz
-mv cloudsoft-amp-karaf-* cloudsoft-amp-karaf
+sudo dpkg -i cloudsoft-amp-karaf-all.deb
 
 echo "Configure AMP Properties"
-mkdir -p /home/vagrant/.brooklyn
-cp /vagrant/files/brooklyn.properties /home/vagrant/.brooklyn/
-chmod 600 /home/vagrant/.brooklyn/brooklyn.properties
+sudo mkdir -p /opt/amp/.brooklyn
+sudo chown amp:amp /opt/amp/.brooklyn
+sudo cp /vagrant/files/brooklyn.properties /opt/amp/.brooklyn/
+sudo chown amp:amp /opt/amp/.brooklyn/brooklyn.properties
+sudo chmod 600 /opt/amp/.brooklyn/brooklyn.properties
 
 echo "Configure MOTD"
 sudo cp /vagrant/files/motd /etc/motd
 
-echo "Installing Karaf wrapper service"
-cd /home/vagrant/cloudsoft-amp-karaf
-echo ".. starting karaf manually"
-./bin/start
-echo ".. wait for karaf to start"
-while ! ./bin/client version > /dev/null 2>&1; do
-  echo ".... waiting for 5 seconds"
-  sleep 5
+echo "Waiting for AMP to start..."
+while ! (sudo grep "BundleEvent STARTED - org.apache.brooklyn.karaf-init" /opt/amp/log/amp.debug.log) > /dev/null ; do
```

Comment:
@aledsage This line fails (but overall install is ok) as the file is not created at this point (I think)

`==> amp: grep: /opt/amp/log/amp.debug.log
==> amp: : No such file or directory`

Also, FYI, users access the vm as the 'vagrant' user but that user cannot access the log files (as restricted to 'amp'). Assuming I'll just update docs to prefix with 'sudo'?